utility/install_chrome.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash 
 
+# This file installs the given CHROME_VERSION to the docker container downloading 
+# a stable version to the dockerfile image
```

Comment:
I'm finding this hard to grok. Wouldn't it be better to say that it installs chrome rather than CHROME_VERSION? I also don't understand what "downloading a stable version to the dockerfile image" means. It sounds like it will install the version specified by CHROME_VERSION _and_ a stable version, which isn't what it's doing. Maybe you can reword this to say it's installing chrome, and choosing  either the stable version, or the version defined by CHROME_VERSION.